Makefile

```diff
@@ -19,6 +19,7 @@ ANTCTL_BINARY_NAME ?= antctl
 OVS_VERSION        := $(shell head -n 1 build/images/deps/ovs-version)
 GO_VERSION         := $(shell head -n 1 build/images/deps/go-version)
 CNI_BINARIES_VERSION := $(shell head -n 1 build/images/deps/cni-binaries-version)
+BGP_VERSION	   := $(go list -m all | grep github.com/osrg/gobgp/v3 | cut -d " " -f2 | cut -c 2-)
```

Comment:
I think it would be better to only have this inside the Dockerfile as I suggested in the original comment. The 2 reasons would be that it is only needed for the docker build and that this way we don't introduce a hard dependency that go is available on the build system.